files/build_templates/docker_image_ctl.j2

```diff
@@ -74,6 +74,7 @@ function postStartAction()
 {
 {%- if docker_container_name == "database" %}
     if [ "$DEV" ]; then
+        docker exec -i database$DEV sysctl -w net.ipv6.conf.all.disable_ipv6=0
```

Comment:
why database only here? is database container different from other containers?

Reply:
The namespaces are created when the the database container is created. So enabling this in the database container ensures it is enabled in the each container in the namespace.

Reply:
Sample output
`root@NAMESPACE5:/home/admin# docker exec -it swss1 bash
root@NAMESPACE5:/# cat /proc/sys/net/ipv6/conf/all/disable_ipv6
0
root@NAMESPACE5:/# exit
root@NAMESPACE5:/home/admin# ip netns exec asic1 cat /proc/sys/net/ipv6/conf/all/disable_ipv6

0
root@NAMESPACE5:/home/admin#`